README.md

```diff
@@ -86,7 +86,8 @@ After you're finished please remove all the comments and instructions!
 - **[ Has posts and users controllers ]**
 - **[ RSpecs for controler actions using request testing ]**
 - **[ Views for User and Post controllers to display content of data base to the user ]**
-
+- **[ Form to add post and comments ]**
+- **[ Contains comment controller ]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
![image](https://github.com/devalibello/blog-app/assets/93039359/cb3bc3d3-9178-4663-ad48-3abf5d3d039f)

- [ ] It would be nice if you fix the highlighted links shown in the image to make your readme descriptive and professional 👀 because these are broken 😨 Kindly fix these links or remove them if you don't have any dedicated section for these broken links 🤔